trade-finance-manager-ui/templates/utilisation-reports/partials/premium-payments-html.njk

```diff
@@ -0,0 +1,132 @@
+{% if premiumPayments.showMatchSuccessNotification %}
+  {% set notificationHtml %}
+    <h3 class="govuk-notification-banner__heading" data-cy="match-success-notification-heading">
+      Match payment recorded
+    </h3>
+    <p class="govuk-body" data-cy="match-success-notification-message">The fee(s) are now at a Match state. Further payments cannot be added to the fee record.</p>
+  {% endset %}
+
+  <div class="govuk-grid-row govuk-!-padding-top-7" data-cy="match-success-notification">
+    <div class="govuk-grid-column-one-half-from-desktop">
+      {{ govukNotificationBanner({
+        html: notificationHtml,
+        type: "success"
+      }) }}
+    </div>
+  </div>
+{% endif %}
+
+{% if premiumPayments.tableDataError or premiumPayments.filterError %}
+  {{ govukErrorSummary({
+    titleText: "There is a problem",
+    errorList: [premiumPayments.tableDataError, premiumPayments.filterError],
```

Comment:
Are we able to move this to the controller? Not keen on array construction in nunjucks.

Can be done in `utilisation-report-reconciliation-for-report.njk` as well, then we have no array consturction in nunjucks

Reply:
this has just been moved, I think it will be within the scope of this PR

Reply:
Understood, but whilst we're here, should only take 2mins to add `errorList` in the controller?

Otherwise can we create a ticket/subtask to address this afterwards please

Reply:
https://ukef-dtfs.atlassian.net/browse/FN-3697 created subtask, it will be a bit more complex given how it is set up